lib/plugins/aws/package/compile/functions.js

```diff
@@ -102,18 +103,19 @@ class AwsCompileFunctions {
       _.get(functionObject, 'package.artifact') ||
       _.get(this, 'serverless.service.package.artifact');
 
-    const regex = new RegExp('s3\\.amazonaws\\.com/(.+)/(.+)');
-    const match = artifactFilePath.match(regex);
-
-    if (!match) return;
+    const s3Object = parseS3URI(artifactFilePath);
+    if (!s3Object) {
+      this.serverless.cli.log(`${artifactFilePath} is not a S3 URI`);
```

Comment:
I don't think we should log something like that as in most cases the `package.artifact` won't be an S3 URI but e.g. a reference to local file and we don't want to log in such cases for sure

Reply:
Ah yeah, sorry. I had forgotten that the `artifactFilePath` is primarily a local path as that isnt my usecase. I'll get rid of the log statement. Hopefully your function that processes a local file path has error handling for when it encounters an invalid local file path.